### PR TITLE
Update kube version and OneClickDemo

### PR DESCRIPTION
**Pull request by aradhalevy**
update the Kubernetes version on requirements.env and on OneClickDemo.

also some more updates to OneClickDemo to make it more compatible with LocalStack and behave similarly to QuickStart.

### CODE REVIEW

File: samples/OneClickDemo/OneClickDemo-Katalog.sh [hunk 1]
```diff
@@ -264,9 +264,10 @@ export SECRET_KEY=1234
 export ENDPOINT="http://127.0.0.1:4566"
 export BUCKET="demo"
 export OBJECT_KEY="sample.csv"
-bin/aws configure set aws_access_key_id ${ACCESS_KEY} 
-bin/aws configure set aws_secret_access_key ${SECRET_KEY}
-bin/aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET}
+export REGION=theshire
+bin/aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY}
```
Comment (roytman) [inline samples/OneClickDemo/OneClickDemo-Katalog.sh:268]: can you separate these 2 commands in different lines.

File: samples/OneClickDemo/OneClickDemo-OMD.sh [hunk 2]
```diff
@@ -258,21 +258,22 @@ bin/kubectl config set-context --current --namespace=fybrik-notebook-sample
...
+export ACCESS_KEY=1234
+export SECRET_KEY=1234
 export ENDPOINT="http://127.0.0.1:4566"
 export BUCKET="demo"
 export OBJECT_KEY="sample.csv"
-bin/aws configure set aws_access_key_id ${ACCESS_KEY} 
-bin/aws configure set aws_secret_access_key ${SECRET_KEY}
-bin/aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET}
+export REGION=theshire
+bin/aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY}
```
Comment (roytman) [inline samples/OneClickDemo/OneClickDemo-OMD.sh:274]: the same as for Katalog

**Review by roytman — COMMENTED** [review]
2 small comments

File: samples/OneClickDemo/OneClickDemo-Katalog.sh (hunk 1 above)
Comment (aradhalevy) [reply samples/OneClickDemo/OneClickDemo-Katalog.sh]: Done

File: samples/OneClickDemo/OneClickDemo-OMD.sh (hunk 2 above)
Comment (aradhalevy) [reply samples/OneClickDemo/OneClickDemo-OMD.sh]: Done

**Review by roytman — APPROVED** [review]

**Review by roytman — COMMENTED** [review]
I tried to run it and the OPA server could not start.
In addition, if it doesn't support MacOS, please print an error message and exit after OS resolving.

**Review by aradhalevy — COMMENTED** [review]
@roytman I simplified the error message if not using Linux as we discussed, now it is ready

**Review by roytman — COMMENTED** [review]
When I run it on Mac, I get the following message:

> cut: illegal option -- -
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-w | -d delim] [file ...]
       
please start the script from the OS validation, the version processing can be after

**Pull request merged**